fh-messaging/lib/services/mbaas_metrics.js

```diff
@@ -213,7 +214,9 @@ module.exports = function(mongodb) {
 
       var options = {query : q, out: {reduce: metric, db: dbName}, scope: {ts: from}};
       logger.info("Rollup options = " + JSON.stringify(options));
+      console.log('@@@@@@@@dfuishf', options);
       collection.mapReduce(mReduceConf.map, mReduceConf.reduce, options, function(err,results) {
+        console.log('@@@@@results', results);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)